Ginger/GingerCore/Drivers/SeleniumDriver.cs

```diff
@@ -3747,6 +3759,19 @@ private ObservableList<ElementInfo> GetAllElementsFromPage(string path, List<eEl
             return foundElementsList;
         }
 
+        private HTMLElementInfo GetElementInfo(string path, HtmlNode htmlElemNode, Tuple<string, eElementType> elementTypeEnum, IWebElement elemObject)
+        {
```

Comment:
comment out GetElementInfoWithIWebElement() if not used anymore

Reply:
GetElementInfoWithIWebElement() function commented !